ref/config/serverConfiguration.adoc

```diff
@@ -221,37 +175,28 @@ operator can be one of `+`, `-`, `*`, `/` for example:
 <variable name="threeagain" value="${six/two}" />
 ----
 
-There are a number of predefined variables:
+The following predefined variables are supported:
 
-* `wlp.install.dir` - the location where the Liberty runtime is installed.
-* `wlp.server.name` - the name of the server.
-* `wlp.user.dir` - the location of the `usr` folder. Defaults to
+* `wlp.install.dir`: the location where the Open Liberty runtime is installed.
+* `wlp.server.name`: the name of the server.
+* `wlp.user.dir`: the location of the `usr` folder. The default is
   `${wlp.install.dir}/usr`.
-* `shared.app.dir` - the location of shared applications. Defaults to
+* `shared.app.dir`: the location of shared applications. The default is
   `${wlp.user.dir}/shared/apps`.
-* `shared.config.dir` - the location of shared configuration files. Defaults to
+* `shared.config.dir`: the directory that contains the server config. The default is
   `${wlp.user.dir}/shared/config`.
-* `shared.resource.dir` - the location of shared resource files. Defaults to
+* `shared.resource.dir`: the location of shared resource files. The default is
```

Comment:
shared.config.dir is referenced as a directory over location, but shared.resource.dir is sticking with location?

Reply:
I think that was an error. Updated.

Reply:
Updated again